azerothcore/azerothcore-wotlk
# fix(DB/quest): Removes deprecated breadcrumb quests as prerequisites fo…

…r 2 Alterac Valley quests



## Changes Proposed:
Removes NextQuestIDs for quests [Speak with Prospector Stonehewer](https://tbc.wowhead.com/quest=7221/speak-with-prospector-stonehewer) and [Speak with Voggah Deathgrip](https://tbc.wowhead.com/quest=7222/speak-with-voggah-deathgrip). These are two deprecated breadcrumb quests that are unavailable in 3.3.5 (no quest starters listed, no allowed races) but each had the NextQuestID set for the followup quests. (Which are [The Battle of Alterac](https://tbc.wowhead.com/quest=7141/the-battle-of-alterac) and [The Battle for Alterac](https://tbc.wowhead.com/quest=7142/the-battle-for-alterac) respectively.) This was incorrectly stopping the two followup quests from being accepted.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9018
- Closes https://github.com/chromiecraft/chromiecraft/issues/2277

## SOURCE:

https://tbc.wowhead.com/quest=7141/the-battle-of-alterac
https://tbc.wowhead.com/quest=7142/the-battle-for-alterac
https://wowpedia.fandom.com/wiki/Speak_with_Prospector_Stonehewer - "The subject of this article or section did not make it out of the patch 1.8.0 test realms."
https://wowpedia.fandom.com/wiki/Speak_with_Voggah_Deathgrip - likewise.

## Tests Performed:

- Ran SQL, no errors/warnings, 2 rows changed as expected.
- Checked in-game, both main quests are now available:

![WoWScrnShot_110721_135133](https://user-images.githubusercontent.com/81782124/140631325-75a88e8c-dbd9-4692-a576-5bd1fb9ab6be.jpg)
![WoWScrnShot_110721_135145](https://user-images.githubusercontent.com/81782124/140631326-73afedf9-5291-490b-965a-8d62989a43bc.jpg)

## How to Test the Changes:


For Horde: .go c 16578 - see if 'Battle for Alterac' is available.
For Alliance: .go c 16991 - see if 'Battle of Alterac' is available.

## Known Issues and TODO List:


N/A.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
